test/jdk/java/awt/print/PrinterJob/JobName/PrinterJobName.java

```diff
@@ -21,173 +21,55 @@
  * questions.
  */
 
-/**
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterJob;
+
+/*
  * @test
  * @bug 4205601
  * @summary setJobName should be used by PrinterJob
  * @key printer
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
  * @run main/manual PrinterJobName
  */
-
-import java.awt.*;
-import java.awt.print.*;
-
 public class PrinterJobName implements Printable {
+    private static final String THE_NAME = "Testing the Job name setting";
+
+    private static final String INSTRUCTIONS =
+            "This test prints a page with a banner/job name of\n\n" +
+            THE_NAME;
+
+    public static void main(String[] args) throws Exception {
+        if (PrinterJob.lookupPrintServices().length == 0) {
+            throw new RuntimeException("Printer not configured or available.");
+        }
+
+        PassFailJFrame passFailJFrame = new PassFailJFrame.Builder()
```

Comment:
```suggestion
        PassFailJFrame passFailJFrame = PassFailJFrame.builder()
```
You have to use the `builder()` method.